tests/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright 2022, Slack Technologies, LLC. All rights reserved.
```

Comment:
Not sure which tests should be written for this, are there some examples that I could follow?

Reply:
Since we don't have the standard way to write unit tests in Python yet, you can skip writing tests this time.